### PR TITLE
Adds orchestration to bringup files.

### PR DESCRIPTION
**Pull request by barulicm**
This PR modifies the bringup files to use the new mission_orchestration package in stsl. This package coordinates all of the individual robot behaviors needed for the game via a behavior tree. To see the default behavior for the game (at least as much of it as currently exists) see [default_mission_tree.xml](https://github.com/RoboJackets/stsl/blob/main/mission_orchestration/behavior_trees/default_mission_tree.xml) in stsl.

I also fixed a problem with some of the other launch files where I had mistakenly used `param` tags instead of `arg` tags when including other launch files.

### CODE REVIEW

File: rj_training_bringup/launch/projects/color_detection.launch.xml
Comment (JasonGibson274) [inline rj_training_bringup/launch/projects/color_detection.launch.xml:6]: consistency in naming?

**Review by JasonGibson274 — APPROVED** [review]

File: rj_training_bringup/launch/projects/color_detection.launch.xml
Comment (barulicm) [reply rj_training_bringup/launch/projects/color_detection.launch.xml:6]: intentional. discussed offline

**Comment by barulicm** [comment]
Will merge once the target branch (planning) is merged into main.

**Pull request merged**
